### PR TITLE
MUI theme and AddItems component styles

### PR DESCRIPTION
**Pull request by dariatsvetkova**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This sets up a Material UI theme alongside the existing global styles and uses both to style the overall layout and the AddItems component. 

*Responsive styles coming soon*

## Related Issue

#14 

## Acceptance Criteria

* Material UI components are used
* /additems page looks like the desktop design in [Figma](https://www.figma.com/file/VH8Z8iouh7F9zBBv1kbuq5/Shopping-App-design?node-id=0%3A1)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓  | :link: Update dependencies |

### Before

screencapture-tcl-22-shopping-list-web-app-additems-2021-05-28-01_25_51

### After

screencapture-localhost-3000-additems-2021-05-28-01_23_50

## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr59-dt-add-items-styles-670l0tmv.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin dt-add-items-styles` and check that branch out with `git checkout origin dt-add-items-styles`
- Run `npm install` to update dependencies, then `npm start` to launch the app in your browser at http://localhost:3000

In the app, navigate to Add Groceries view to see the changes.

### CODE REVIEW

File: src/components/General/Layout.js [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
 
 const StyledMain = styled.main`
   background-image: url(${BackgroundIcons});
-  min-height: 100%;
+  min-height: 100vh;
+`;
+
+const StyledHeader = styled.header`
+  width: 100%;
+  background: #5f9645;
```
Comment (laurenmbeatty) [inline src/components/General/Layout.js:13]: It's a good idea to use the theme. Just import above and do:

```suggestion
  background: ${theme.primary.main};
```

File: src/components/General/globalStyles.js [hunk 2]
```diff
@@ -9,6 +12,21 @@ const GlobalStyle = createGlobalStyle`
...
   }
+  body{
+    margin: 0;
+  }
+  .App {
+    flex: 0 0 50vh;
+    width: 100%;
+    margin: 4.5em;
+    padding: 3.5em;
+    background-color: #FAFAF9;
```
Comment (laurenmbeatty) [inline src/components/General/globalStyles.js:23]:
```suggestion
    background-color: ${theme.background.paper};
```

**Review by ablizben — APPROVED** [review]
Thank you, Daria!!!

**Review by laurenmbeatty — APPROVED** [review]
Love that logo!

**Review by cafloyd — APPROVED** [review]
Amazing, thank you, Daria!!!

**Pull request merged**